app/models/institution_builder.rb

```diff
@@ -535,7 +535,7 @@ def self.build_institution_programs(version_number)
         full_time_undergraduate,
         graduate,
         full_time_modifier,
-        length,
+        length_in_weeks,
```

Comment:
Why this change? This is populating `institution_programs.length_in_hours` and we already have `length_in_weeks` populating `institution_programs.length_in_weeks` in this SQL (see line 551).

Reply:
This is a short term fix to an issue that they aren't sure we should fix

Reply:
Can you add a comment or link to an issue in the code so that other engineers aren't confused about it later?